pcmanfm/desktopwindow.h

```diff
@@ -175,7 +175,8 @@ protected Q_SLOTS:
     bool wallpaperRandomize_;
     QPixmap wallpaperPixmap_;
     Launcher fileLauncher_;
-    bool showWmMenu_;
+    bool passButtonsToRoot_;
```

Comment:
We'll get a FTBFS here; please restore `showWmMenu_` and remove `passButtonsToRoot_`.

Reply:
Done.